diff -Naru /var/www/command/worker.php www/command/worker.php
--- /var/www/command/worker.php	2019-04-14 00:59:03.238125950 +0200
+++ www/command/worker.php	2019-04-28 07:59:38.811966407 +0200
@@ -405,6 +405,7 @@
 workerLog('worker: Configure MPD outputs');
 $mpdoutput = configMpdOutputs();
 sysCmd('mpc enable only ' . $mpdoutput);
+sysCmd('mpc enable mpd_oled_FIFO');
 setMpdHttpd();
 // report mpd outputs
 sendMpdCmd($sock, 'outputs');
diff -Naru /var/www/inc/playerlib.php www/inc/playerlib.php
--- /var/www/inc/playerlib.php	2019-04-14 01:01:35.957370609 +0200
+++ www/inc/playerlib.php	2019-04-28 08:03:20.133466875 +0200
@@ -1295,6 +1295,9 @@
 	$data .= "always_on \"yes\"\n";
 	$data .= "}\n";
 
+        // mpd_oled FIFO
+        $data .= file_get_contents('/usr/local/etc/mpd_oled_fifo.conf');
+
 	$fh = fopen('/etc/mpd.conf', 'w');
 	fwrite($fh, $data);
 	fclose($fh);
